tests/webapp/api/test_performance_data_api.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from django.core.urlresolvers import reverse
+from rest_framework.test import APIClient
+
+from treeherder.perf.models import PerformanceSignature
+
+
+@pytest.fixture
+def summary_perf_signature(test_perf_signature):
+    # summary performance signature don't have test value
+    signature = PerformanceSignature.objects.create(
+        repository=test_perf_signature.repository,
+        signature_hash=(40*'s'),
+        framework=test_perf_signature.framework,
+        platform=test_perf_signature.platform,
+        option_collection=test_perf_signature.option_collection,
+        suite='mysuite'
+    )
+    signature.subtests.add(test_perf_signature)
+    return signature
+
+
+def test_no_summary_performance_data(webapp, test_perf_signature,
+                                     jm):
+
+    resp = webapp.get(reverse('performance-signatures-list',
+                              kwargs={"project": jm.project}))
+    assert resp.status_int == 200
+
+    # we should get only one signature date and it
+    # is a no summary test.
+    assert resp.json.get('subtests', None) is None
+    assert len(resp.json.keys()) == 1
+    assert resp.json.keys()[0] == test_perf_signature.signature_hash
+    # we don't have test_options and the lower_is_better because they haven't been
+    # init when we generate test_perf_signature mock.
```

Comment:
This comment isn't true (these get default values if not specified). :) It's also unnecessary. I think we can remove it.